pkg/drain/drainer.go

```diff
@@ -131,6 +145,28 @@ func (d *Drainer) CompleteDrainNode(ctx context.Context, node *corev1.Node) (boo
 	return completed, nil
 }
 
+// removeDaemonSetsFromNode go over all the remain pods and search for DaemonSets that have SR-IOV devices to remove them
```

Comment:
so setting `IgnoreAllDaemonSets: false`  in drain.Helper, will delete/evict daemonset pods ?